generators/server/templates/gradle.properties.ejs

```diff
@@ -66,7 +66,7 @@ jaxb_runtime_version=2.3.2
 
 # gradle plugin version
 micronaut_plugin_version=1.4.5
```

Comment:
Should this be updated too?

Reply:
I have no idea. But I can try to see if it succeeds or fails